core/src/main/java/com/linecorp/armeria/client/limit/DefaultConcurrencyLimit.java

```diff
@@ -52,6 +57,11 @@
 
     DefaultConcurrencyLimit(Predicate<? super ClientRequestContext> predicate,
```

Comment:
I found out that this constructor is only called from tests. How about removing this constructor and calling the below constructor directly?